.github/workflows/python-quality.yml

```diff
@@ -23,20 +23,26 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - name: Install dependencies
+
+      # Setup venv
+      # TODO: revisit when https://github.com/astral-sh/uv/issues/1526 is addressed.
+      - name: Setup venv + uv
         run: |
-          pip install --upgrade pip
-          pip install .[dev]
-      - run: ruff check tests src contrib # linter
-      - run: ruff format --check tests src contrib # formatter
-      - run: python utils/check_contrib_list.py
-      - run: python utils/check_static_imports.py
-      - run: python utils/generate_async_inference_client.py
-      - run: python utils/generate_inference_types.py
+          pip install --upgrade uv
+          uv venv
+
+      - name: Install dependencies
+        run: uv pip install "huggingface_hub[dev] @ ."
```

Comment:
You _should_ be able to do `uv pip install --system "huggingface_hub[dev] @ ."` on the latest release, if you want to remove the virtualenv (per the PR summary), but using a virtualenv is also good.

Reply:
Thanks @charliermarsh! I just saw the issue has been merged on uv's repo :tada:. Thanks for the direct ping here :heart:

Reply:
Of course! It's pretty new so of course ping if you see issues with it.